css/style.css

```diff
@@ -0,0 +1,43 @@
+* {
+}
+body {
+}
+
+#title,
+#description {
+  text-align: center;
+}
+#title {
+  margin-top: 10vh;
+}
+#description {
+  font-size: 24px;
+}
+.form-container {
+  width: 50%;
+  background-color: #2d2945;
+  color: white;
+  margin: 5vh auto;
+}
```

Comment:
it is best practice to have a new line after each class selector. Please add new line below this class selector